index.html

```diff
@@ -1,49 +1,219 @@
+              <p>See Project</p>
+            </div>
+          </div>
+        </div>
+        <div class="work-2">
+          <div class="work2-content">
+            <h1>Profesional Art Printing Data</h1>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li class="list2">html</li>
+              <li class="list2">bootstrap</li>
+              <li class="list2">Ruby</li>
+            </ul>
+            <div class="list2 work2-button">
+              <p>See Project</p>
+            </div>
+          </div>
+        </div>
+        <div class="work-2">
+          <div class="work2-content">
+            <h1>Profesional Art Printing Data</h1>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li class="list2">html</li>
+              <li class="list2">bootstrap</li>
+              <li class="list2">Ruby</li>
+            </ul>
+            <div class="list2 work2-button">
+              <p>See Project</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="about-me-section">
+        <div class="about-me-1">
+          <div class="about-me1-content">
+            <h1>About me</h1>
+            <p>
+              Hello I`m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don`t
+              hestiate to contact me.
+            </p>
+            <button>Get My Resume</button>
+          </div>
+          <div class="about-me1-separation">
+            <div class="line2"></div>
+          </div>
+        </div>
+        <div class="about-me-2">
+          <div class="topic">
+            <img src="./svg/language.svg" alt="">
+            <p>Language</p>
+          </div>
+          <ul class="discussion">
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <div class="topic">
+            <img src="./svg/frameworks.svg" alt="">
+            <p>Frameworks</p>
+          </div>
+          <ul class="discussion">
+            <li>Bootstrap</li>
+            <li>Ruby On Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+          <div class="topic">
+            <img src="./svg/skills.svg" alt="">
+            <p>Skills</p>
+          </div>
+          <ul class="discussion">
+            <li>CodeKit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>GitLab</li>
+            <li>Terminal</li>
+          </ul>
```

Comment:
- Kindly make sure to add `a tag` for all `li tags` in the project.

Reply:
or <a> tag?